packages/components-react/mark-react/package.json

```diff
@@ -32,6 +32,7 @@
     "build": "rollup --config ./rollup.config.mjs",
     "clean": "rimraf ./dist/ ./*.tsbuildinfo ./.rollup.cache/",
     "test": "jest",
+    "test:watch": "jest --watch",
```

Comment:
zelfde vraag als in andere PR: is `watch:test` niet handiger?

Reply:
Zelfde antwoord ook :D